src/style.css

```diff
@@ -2,0 +1,49 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src:
+    url(fonts/Avenir-Book.ttf) format('truetype'),
+    url(fonts/Avenir-Heavy.ttf) format('truetype'),
+    url(fonts/Avenir.ttc) format('opentype');
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar--big {
+  padding-top: 25px;
+  padding-left: 62px;
+  padding-bottom: 26px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+  margin: 20px 0;
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margins, but not both, to avoid potential margin collapse issues.